README.md

```diff
@@ -1,2 +1,86 @@
-# the-project
-The Project is a sample project we ask prospective consultants to complete as part of their interview process. 
+# The Project
+
+## Introduction
+
+`The Project` is a self-guided application project for all prospective Lofty consultants. We believe that an engineer's merit is not always described by their resumé. Sometimes a great engineer hasn't had an opportunity to prove themselves yet or maybe they're just uncomfortable in interviews and don't end up giving their true ability the chance to shine in an stressful interview. `The Project` helps us determine a potential consultant's true ability outside of the black and white confines of their resumé and sometimes those awkward face-to-face interviews.
+
+`The Project` will explore your capability as a Python developer using our preferred tooling. We don't expect all applicants to have proficiency in all of the technologies specified in this document.  Lofty Labs Consultants often work independently with only a small amount of direct supervision and often work directly on projects built by other engineering teams as well. Sometimes this introduces new systems to us that we haven't used before. This project is meant to expose you to the same situation wherein maybe you haven't used all of the technologies or tooling that we have specified in the requirements. Completion of `The Project` will show that even if you don't have experience with all of the specifications listed, you're a dedicated and passionate engineer that is able to pick up new systems and technologies at an at least basic level. The ability to adapt to a new and possibly unknown technology is a core competency of all Lofty Engineering Consultants.
+
+## Getting Help
+
+While working on this project, you will have access to members of the Lofty Labs consulting team via Slack. Contact us at `jobs@hirelofty.com` and let us know you'd like to be a part of the project and we'll add you to our slack channel named `#the-academy`. If you get stuck, or even just want to discuss the project in general, you are encouraged to ask questions and get help from the Lofty Engineering team as well as the other potential consultants working on the same project as you. A key component of the value we add to our client's projects as a consultantcy is our ability to communicate across many disparate teams and utilize everyone’s particular areas of expertise. Our clients don't hire us because we know every solution off-hand; our client's hire Lofty Labs because we know how to solve problems. Solving our client's problems includes research, continuing education, and cross-team knowledge sharing. What this means for you as a potential consultant working on `The Project` is that we don't mind you asking questions throughout your progress in slack. We don't care that you ask questions. We care that you ask *the right* questions.
```

Comment:
yep i was just thinking how we could get people in the slack room and thought this might be quickest.